.travis.yml

```diff
@@ -6,3 +8,14 @@ git:
 script:
   - tests/phplint.sh
   - tests/strict_types.sh
+  - composer install
+  - mv .env.sample .env
+  - mv config/config.specific.sample.php config/config.specific.php
+  - mv config/SmrMySqlSecrets.sample.inc config/SmrMySqlSecrets.inc
+  - mv config/discord/config.specific.sample.php config/discord/config.specific.php
+  - mv config/irc/config.specific.sample.php config/irc/config.specific.php
+  - mv config/npc/config.specific.sample.php config/npc/config.specific.php
+  - docker-compose run mysql-integration-test
```

Comment:
Maybe these need to be run in detached mode, i.e. with `--detach`? Otherwise I think they will block and prevent the remaining commands from being executed.

Reply:
Yeah I fixed them. I was using `docker-compose run` instead of `up -d`.